src/openssl/hmac.cr

```diff
@@ -13,27 +14,9 @@ class OpenSSL::HMAC
   #
   # It may contain non-ASCII bytes, including NUL bytes.
   #
-  # *algorithm* is a `Symbol` of a supported digest algorithm:
-  # * `:md4`.
-  # * `:md5`.
-  # * `:ripemd160`.
-  # * `:sha1`.
-  # * `:sha224`.
-  # * `:sha256`.
-  # * `:sha384`.
-  # * `:sha512`.
-  def self.digest(algorithm : Symbol, key, data) : Bytes
-    evp = case algorithm
-          when :md4       then LibCrypto.evp_md4
-          when :md5       then LibCrypto.evp_md5
-          when :ripemd160 then LibCrypto.evp_ripemd160
-          when :sha1      then LibCrypto.evp_sha1
-          when :sha224    then LibCrypto.evp_sha224
-          when :sha256    then LibCrypto.evp_sha256
-          when :sha384    then LibCrypto.evp_sha384
-          when :sha512    then LibCrypto.evp_sha512
-          else                 raise "Unsupported digest algorithm: #{algorithm}"
-          end
+  # *algorithm* is of type `OpenSSL::Algorithm`.
```

Comment:
No need to precise the type here in the documentation, you could even not document it at all, it's pretty self explanatory!  If you still want to document this argument, say what it is for / what it'll do

Reply:
Ok, i've removed this comment